<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>найти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shirota_line">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="shirota_label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>77</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>широта</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dolgota_line">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="dolgota_label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>77</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>долгота</string>
   </property>
  </widget>
  <widget class="QLabel" name="karta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>631</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
